packages/loopring_v3/contracts/impl/Exchange.sol

```diff
@@ -230,8 +272,8 @@ contract Exchange is IExchange, Claimable, ReentrancyGuard
         address tokenAddress
         )
         external
-        nonReentrant
```

Comment:
I'd like to keep the order like this everywhere:
```
nonReentrant
onlyOwner
```
This makes it much easier to test non-reentrancy. The tests will use the ABI file to find all `external` functions and will test them automatically for reentrancy. If `onlyOwner` (or some other modifier) is the first modifier the test will fail because `msg.sender != owner` (the reentrancy is tested from a test contract).

Reply:
great, I'll change it.